vnext/ReactUWP/Modules/ThemingModuleUwp.cpp

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "ThemingModuleUwp.h"
+#include <iomanip>
+#if _MSC_VER <= 1913
+// VC 19 (2015-2017.6) cannot optimize co_await/cppwinrt usage
+#pragma optimize( "", off )
+#endif
+
+namespace winrt {
+  using namespace Windows::UI::Xaml;
+}
+
+namespace react {
+  namespace uwp {
+
+  //
+  // Theming
+  //
+
+  Theming::Theming(const std::shared_ptr<IReactInstance>& reactInstance, const std::shared_ptr<facebook::react::MessageQueueThread>& defaultQueueThread) : react::windows::PlatformTheme()
+    , m_wkReactInstance(reactInstance)
+    , m_queueThread(defaultQueueThread)
+  {
+    m_currentTheme = winrt::Application::Current().RequestedTheme();
+    m_isHighContrast = m_accessibilitySettings.HighContrast();
+    m_RGBValues = getHighContrastRGBValues();
+
+    m_highContrastChangedRevoker = m_accessibilitySettings.HighContrastChanged(winrt::auto_revoke,
+      [this](const auto&, const auto&) {
+        folly::dynamic eventData = folly::dynamic::object("highContrastRGBValues", getHighContrastRGBValues())
+          ("isHighContrast", getIsHighContrast());
+        fireEvent("highContrastChanged", std::move(eventData));// fire event with information that it's a high contrast change and what the scheme is
+    });
+
+    m_colorValuesChangedRevoker = m_uiSettings.ColorValuesChanged(winrt::auto_revoke,
+      [this](const auto&, const auto&) {
+        m_queueThread->runOnQueue([this]() {
+          if (m_currentTheme != winrt::Application::Current().RequestedTheme())
+          {
+            m_currentTheme = winrt::Application::Current().RequestedTheme();
+            folly::dynamic eventData = folly::dynamic::object("currentTheme", getCurrentTheme());
+            fireEvent("appThemeChanged", std::move(eventData));
```

Comment:
>appThemeChanged [](start = 23, length = 15)

look like it's a dupe of [https://github.com/microsoft/react-native-windows/pull/2684](https://github.com/microsoft/react-native-windows/pull/2684)